src/main/java/worldofzuul/PresentationLayer/Grid.java

```diff
@@ -133,7 +133,7 @@ private void drawGrid() {
 
         //Then draw the background
         gc.save();
-        gc.translate((int) (windowWidth / 2 - (gameWidth / 2)),
+        gc.translate((int) (windowWidth / 2 - (gameWidth / 2)), //forklar tak
                 (int) (windowHeight / 2 - (gameHeight / 2)));
```

Comment:
Translate flytter 0,0 punktet for det koordinatsystem jeg tegner i. 
Så hvis jeg bruger `gc.translate(100,100)`, så flytter jeg mit koordinatsystem 100 pixels til højre, og 100 pixels ned på siden. Det betyder at hvis jeg så siger `gc.drawLine(0,0,100,100)` _(som normalt tegner en linje fra (0,0) til (100,100))_, så tegner jeg i virkeligheden en linje fra (100,100) til (200,200), fordi det hele er forskudt/translatet med 100,100.

Skal nok prøve at forklare det igen på en anden måde, men håber det gav lidt indblik.